src/app-layer-rdp-rust.c

```diff
@@ -0,0 +1,42 @@
+/* Copyright (C) 2019 Open Information Security Foundation
```

Comment:
Another nit -- Is it necessary to embed `rust` within the filename?

Reply:
No. We should probably get rid of this naming convention. It made sense with DNS when we had a C and a Rust one.  But it doesn't make sense anymore.